hedera-node/src/main/java/com/hedera/services/txns/ethereum/EthereumTransitionLogic.java

```diff
@@ -194,12 +194,15 @@ public Function<TransactionBody, ResponseCodeEnum> semanticCheck() {
 	public void preFetch(final TxnAccessor accessor) {
 		var ethTxData = spanMapAccessor.getEthTxDataMeta(accessor);
 
-		EthereumTransactionBody op = accessor.getTxn().getEthereumTransaction();
-		ethTxData = maybeUpdateCallData(accessor, ethTxData, op);
+//		EthereumTransactionBody op = accessor.getTxn().getEthereumTransaction();
+//		ethTxData = maybeUpdateCallData(accessor, ethTxData, op);
```

Comment:
Were those two left to be like that on purpose even though they have been moved a couple of lines down?